README.md

```diff
@@ -63,7 +63,7 @@ In case this doesn't happen, navigate to:
 
 * [Icons Customization](https://github.com/vscode-icons/vscode-icons/wiki/Customization)
 * [Project Auto-Detection](https://github.com/vscode-icons/vscode-icons/wiki/Pad)
-* [Custom Configuration](https://github.com/vscode-icons/vscode-icons/wiki/Confguration)
+* [Customization](https://github.com/vscode-icons/vscode-icons/wiki/Customization)
```

Comment:
The correct page is https://github.com/vscode-icons/vscode-icons/wiki/Configuration.
The issue is a typo in the name not the link.

Reply:
good point

Reply:
No. `Custom Configuration` refers in configuring the `vscode` settings.

Reply:
@paulvi, could you be so kind to update the link so we can merge this? Thank you very much!